### PR TITLE
Bulk upload initial write

### PR DESCRIPTION
**Pull request by nathancashmoredwp**
Provides scripts to create and upload bulk seed data.

### CODE REVIEW

File: scripts/seeds/knexfile-chef.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    },
+  },
+
+  production: {
+    client: 'pg',
+    connection: process.env.DATABASE_URL,
+    seeds: {
+      directory: './chef',
+    },
+  },
```
Comment (lukasz-kaniowski) [inline scripts/seeds/knexfile-chef.js]: can we reuse the config that we already have. i.e. 
```
const config = require('../knexfile.js')
....

production: { Object.assign(
{  seeds: {
     directory: './chef',
  }
},
config.production
)

File: scripts/template/knexfile.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    },
+  },
+
+  production: {
+    client: 'pg',
+    connection: process.env.DATABASE_URL,
+    seeds: {
+      directory: './-dir-',
+    },
+  },
```
Comment (lukasz-kaniowski) [inline scripts/template/knexfile.js]: same as above

**Comment by lukasz-kaniowski** [comment]
can we use .csv file instead of .ods?

File: scripts/template/knexfile.js (hunk 2 above)
Comment (nathancashmoredwp) [inline scripts/template/knexfile.js]: Good question .... will check

**Comment by nathancashmoredwp** [comment]
The .csv is an interim step as the intended output will be the jobs_seed.js file... the ODS is the master which has a number of tabs each representing different use cases.  ODS may become NUMBERS or XLS.

**Pull request merged**